instrumentation/metric/histogram.go

```diff
@@ -28,55 +28,55 @@ type histogramExport struct {
 func newHistogram(name string, max int64) *Histogram {
 	return &Histogram{
 		namedMetric: namedMetric{name: name},
-		histo:       hdrhistogram.New(0, max, 1),
+		histo:       hdrhistogram.NewWindowed(5, max, 1, 3),
```

Comment:
are you rotating it anywhere?

Reply:
My understanding is that it is rotated automatically because it's rolling, so it uses 5 underlying histograms that constantly rotate by themselves.